### ISSUE TITLE
flutter analyze fails on version 1.11.0

### ISSUE THREAD

**Issue by tmorone-rezi** [mwaylabs/flutter-apns#9]
On the beta channel, the latest release (1.11.0) throws this error during flutter analyze:

Couldn't read file LocalFile:
'.../dev/lib/flutter/.pub-cache/git/flutter-apns-7f918932cf8bb9f283c19bab728ac557ff4a1141/android/src/main/kotlin/com/mwaysolutions/flutter/apns/FlutterApnsPlugin.kt' even though it exists. Please verify that this file has read permission and try again.

It is looking for a kotlin file, but there is no kotlin in the project.

**Comment by Alqueraf** [comment]
Hey @tmorone-rezi did you find a solution for this?

**Comment by SinaSeirafi** [comment]
After some searching, I came across this solution: 

I created the file it was looking for and converted the ".java" file to ".kt" using android studio and put it there and It passed that state.

Here's the Kotlin code:

```
package com.mwaysolutions.flutter.apns

import io.flutter.plugin.common.MethodCall
import io.flutter.plugin.common.MethodChannel
import io.flutter.plugin.common.MethodChannel.MethodCallHandler
import io.flutter.plugin.common.MethodChannel.Result
import io.flutter.plugin.common.PluginRegistry.Registrar

/** FlutterApnsPlugin  */
class FlutterApnsPlugin : MethodCallHandler {

    @Override
    fun onMethodCall(call: MethodCall, result: Result) {
        if (call.method.equals("getPlatformVersion")) {
            result.success("Android " + android.os.Build.VERSION.RELEASE)
        } else {
            result.notImplemented()
        }
    }

    companion object {
        /** Plugin registration.  */
        fun registerWith(registrar: Registrar) {
            val channel = MethodChannel(registrar.messenger(), "flutter_apns")
            channel.setMethodCallHandler(FlutterApnsPlugin())
        }
    }
}
```

But I'm not sure if it still works. I Will update this post after testing.

**Comment by tmorone-rezi** [comment]
No code change is necessary.

The file at this location:
android/src/main/java/com/example/flutter_apns/FlutterApnsPlugin.java

Needs to move to this location:
android/src/main/java/com/mwaysolutions/flutter/apns/FlutterApnsPlugin.java

Our fork is not in a mergeable state right now, otherwise I would put in the PR.

**Comment by Alqueraf** [comment]
Thanks, that works 👍

**Comment by szotp** [comment]
Sorry guys, I have been busy with other stuff. I will apply that fix tomorrow.

**Comment by szotp** [comment]
I just released 1.0.5, let me know if all is ok.

**Comment by Alqueraf** [comment]
Updated and works perfectly, thanks 👍
